src/components/items/ItemScreen.js

```diff
@@ -48,136 +42,134 @@ const ItemScreen = () => {
-              aria-controls={`scrollable-prevent-tabpanel-${0}`}
-            />
-            <Tab
-              label="Members"
-              id={`scrollable-prevent-tab-${1}`}
-              aria-controls={`scrollable-prevent-tabpanel-${1}`}
-            />
-            <Tab
-              label="Settings"
-              id={`scrollable-prevent-tab-${2}`}
-              aria-controls={`scrollable-prevent-tabpanel-${2}`}
-            />
-          </Tabs>
-        </AppBar>
-        <TabPanel value={value} index={0}>
-          {children.length === 0 ? (
-            <Typography>No Children found</Typography>
-          ) : (
-            <ItemsTable
+        <div className={classes.root}>
+          <AppBar position="static" color="default">
+            <Tabs
+              value={value}
+              onChange={handleChange}
+              variant="fullWidth"
+              scrollButtons="off"
+              textColor="primary"
+              aria-label="scrollable-prevent-tabs"
+              indicatorColor="primary"
+            >
+              <Tab
+                label="Children"
+                id={`scrollable-prevent-tab-${0}`}
+                aria-controls={`scrollable-prevent-tabpanel-${0}`}
+              />
+              <Tab
+                label="Members"
+                id={`scrollable-prevent-tab-${1}`}
+                aria-controls={`scrollable-prevent-tabpanel-${1}`}
+              />
+              <Tab
+                label="Settings"
+                id={`scrollable-prevent-tab-${2}`}
+                aria-controls={`scrollable-prevent-tabpanel-${2}`}
+              />
+            </Tabs>
+          </AppBar>
+          <TabPanel value={value} index={0}>
+            {children.isEmpty() ? (
+              <Typography>No Children found</Typography>
+            ) : (
+              <ItemsTable
+                empty={false}
+                items={children}
+                id={buildChildrenItemsTableId(itemId)}
+              />
+            )}
+          </TabPanel>
+          <TabPanel value={value} index={1}>
+            <MembersTable
               empty={false}
-              items={List(children)}
-              id={buildChildrenItemsTableId(itemId)}
+              members={members}
+              id={buildMembersTableId(itemId)}
             />
-          )}
-        </TabPanel>
-        <TabPanel value={value} index={1}>
-          <MembersTable
-            empty={false}
-            members={List(members)}
-            id={buildMembersTableId(itemId)}
-          />
-        </TabPanel>
-        <TabPanel value={value} index={2}>
-          <Typography>Extra:</Typography>
-          <ReactJson src={item?.extra} />
-        </TabPanel>
-      </div>
+          </TabPanel>
+          <TabPanel value={value} index={2}>
+            <Typography>Extra:</Typography>
+            <ReactJson src={item.get('extra')} />
+          </TabPanel>
+        </div>
+      </>
```

Comment:
Suggestion: I shouldn't hesitate to cut in parts your component, especially if you add condition clauses inside. You don't necessarily need to create a whole new component, but `renderX()` function could fit well.